server/server.js

```diff
@@ -229,15 +232,44 @@ APP.get('/api/circle/publickey', async (req, res) => {
     }
 });
 
+//webhook for payadmit notifications. URL willhave to be
```

Comment:
when you have more than one line of comments, it is customary to use this format:
```
/**
*
**/
```

Reply:
done